src/components/HandlerManager/HandlerManager.js

```diff
@@ -13,6 +13,7 @@ class HandlerManager extends React.Component {
     modules: PropTypes.shape({
       handler: PropTypes.array,
     }),
+    props: PropTypes.object,
```

Comment:
Wait -- there's a prop called `props`? That's just _asking_ for confusion. Surely better to call it something like `subcomponentProps`?

Reply:
I thought about it, but decided to use the `redux-form` name. It was a choice between what's obvious and memorable and what could be confusing. So since the `props` nomenclature is already in use in stuff we currently use, it broke the tie for me.

Reply:
OK, I guess.